File: Modul_2/Bahasa C/PRAK203-2510817120005-NorKhalisyah.c
#include <stdio.h>

int main() {
    double a, b, i, j, x, y, hasil;
    int count;

    printf("Masukkan 6 nilai\n");
    count = scanf("%lf %lf %lf %lf %lf %lf", &a, &b, &i, &j, &x, &y);

    if (count == 6) {
        hasil = (a - b) * (i / j) - (x + y);
        printf("%.3lf\n", hasil);
    } 
    else if (count == 2) {
        scanf("%lf %lf", &a, &b);
        scanf("%lf %lf", &i, &j);
        scanf("%lf %lf", &x, &y);

        hasil = (a - b) * (i / j) - (x + y);
        printf("%.3lf\n", hasil);
    } 

    return 0;
}

File: Modul_2/Bahasa Python/PRAK203-2510817120005-NorKhalisyah.py
print("Masukkan 6 nilai")
data = input().split()

if len(data) == 2:
    data += input().split()  
    data += input().split()  

a, b, i, j, x, y = map(float, data)

hasil = (a - b) * (i / j) - (x + y)

print(f"{hasil:.3f}")
